scripts/upgrade_admin_server.sh

```diff
@@ -77,6 +77,15 @@ upgrade_admin_server()
     # Signalize that the upgrade correctly ended
     echo "12.2" >> $UPGRADEDIR/admin-server-upgraded-ok
 
+    # delete current OS values
+    knife exec -E "n = nodes.find(:roles => 'provisioner-server').first
+n.target_platform = nil
+n.provisioner.default_os = nil
+n.save"
+
+    # let the provisioner fill the new OS values
+    chef-client
+
```

Comment:
Is the chef-client run here really required. Or could we just explictily set the attributes to the new values? This is somewhat along the lines of @MaximilianMeister comment (https://github.com/crowbar/crowbar/pull/2276#pullrequestreview-6998801)